bin/pic-create

```diff
@@ -79,6 +79,14 @@ if [ $# -eq 2 ] ; then
     src_path=$1
     dst_path=$2
     selected_default_folder_to_copy=$folder_to_clone
+    # Use example directory if no source directory available at current directory
+    if [[ -d $PIC_EXAMPLES/$src_path && ! -d ./$src_path ]]; then
+        src_path=$PIC_EXAMPLES/$src_path
+    fi
+
+    if [[ -d $PIC_EXAMPLES/$src_path && -d ./$src_path ]]; then
+        echo "The local directory will be used instead of the PIConGPU example directory for the source path!" >&2
```

Comment:
suggestion: `warning: 'src_dir' is set to the local directory ./$src_path`

Reply:
While looking again to it maybe it is better to add before the copy starts a message: `clone $dst_path from $src_path`